backend/api.test/EndpointTest.cs

```diff
@@ -179,27 +181,63 @@ private async Task PopulateAreaDb(string installationCode, string plantCode, str
             );
 
             // Act
-            var installationResponse = await _client.PostAsync(installationUrl, installationContent);
-            Assert.NotNull(installationResponse);
-            var plantResponse = await _client.PostAsync(plantUrl, plantContent);
-            Assert.NotNull(plantResponse);
-            var deckResponse = await _client.PostAsync(deckUrl, deckContent);
-            Assert.NotNull(deckResponse);
-            var areaResponse = await _client.PostAsync(areaUrl, areaContent);
-            Assert.NotNull(areaResponse);
+            _ = await _client.PostAsync(installationUrl, installationContent);
```

Comment:
Why were the `Assert.NotNull` removed?

Reply:
This was mainly done since this Post functionality is tested elsewhere so this populate function is treated as a utility function instead of a test. But a different PR does update this function which does include some more assertions.